test/SqlExtension.Tests/TestData.cs

```diff
@@ -1,28 +1,27 @@
-﻿using System;
-
-namespace SqlExtension.Tests
-{
-    #pragma warning disable CS0659 // Type overrides Object.Equals(object o) but does not override Object.GetHashCode()
-    public class TestData
-    #pragma warning restore CS0659
-    {
-        public int ID { get; set; }
-
-        public string Name { get; set; }
-
-        public double Cost { get; set; }
-
-        public DateTime Timestamp { get; set; }
-
-        public override bool Equals(object obj)
-        {
-            var otherData = obj as TestData;
-            if (otherData == null)
-            {
-                return false;
-            }
-            return this.ID == otherData.ID && this.Cost == otherData.Cost && ((this.Name == null && otherData.Name == null) ||
-                string.Equals(this.Name, otherData.Name, StringComparison.OrdinalIgnoreCase)) && this.Timestamp.Equals(otherData.Timestamp);
-        }
-    }
-}
+﻿using System;
+
+namespace SqlExtension.Tests
+{
+    #pragma warning disable CS0659 // Type overrides Object.Equals(object o) but does not override Object.GetHashCode()
+    public class TestData

```

Comment:
Could all of these changes be related to line endings?

Reply:
Seems likely...I assume you see them as `\n` locally?

Reply:
https://github.com/Azure/azure-functions-sql-extension/compare/chlafren/testeroni

Definitely a line ending thing. Personally, I don't care either way. Any strong opinions on how we should enforce this (if at all?)?

Reply:
I think we can remove that - github takes care of that stuff for us generally (and if there are files that are for specific OSs such as .bat files then we can control that separately through gitattributes like vs code does https://github.com/microsoft/vscode/blob/main/.gitattributes)